webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts

```diff
@@ -246,4 +256,8 @@ export class TestCreatePageComponent implements OnInit {
       }
     );
   }
+
+  public isRunnerAvailable(runner: string) : boolean{
+    return this.contracts.includes(runner)
```

Comment:
I think this cannot work as contract types (see the list on https://github.com/microcks/microcks/blob/1.10.x/webapp/src/main/webapp/src/app/models/service.model.ts#L106) are different than runner types (see the list on https://github.com/microcks/microcks/blob/1.10.x/webapp/src/main/webapp/src/app/models/test.model.ts#L83).

Typically, a contract type is `POSTMAN_COLLECTION` but the test runner type is `POSTMAN`. You'll need to initialize a translation map between the two.

Reply:
Here I am passing value of contract type based on the runner option. As we have fixed runner options, I think we can put string value of contract type for the checking directly. You can see this at `line 61` of `test-create.page.html`